### PR TITLE
[AWS] Add CloudFront logs datastream

### PR DESCRIPTION
**Pull request by legoguy1000**
## What does this PR do?

Add CloudFront logs datastream

## Checklist

- [X] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/master/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [X] I have verified that all data streams collect metrics or logs.
- [X] I have added an entry to my package's `changelog.yml` file.
- [X] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

**Comment by legoguy1000** [comment]
@kaiyan-sheng sorry to keep bugging u, but I got another one for ya 😁

File: packages/aws/docs/cloudfront.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+An example event for `cloudwatch` looks as following:
+
+```json
+{
+    "@timestamp": "2020-02-20T07:02:37.000Z",
+    "data_stream": {
+        "namespace": "default",
+        "type": "logs",
+        "dataset": "aws.cloudwatch_logs"
```
Comment (legoguy1000) [inline packages/aws/docs/cloudfront.md:60]: Need to rebuild docs with correct sample event.

File: packages/aws/manifest.yml [hunk 2]
```diff
@@ -535,5 +535,20 @@ policy_templates:
         title: AWS Route53 logo
         size: 32x40
         type: image/svg+xml
+  - name: cloudfront
+    title: AWS CloudFront
+    description: Collect logs from AWS CloudFront with Elastic Agent
+    data_streams:
+      - cloudfront_logs
+    inputs:
+      - type: aws-s3
```
Comment (kaiyan-sheng) [inline packages/aws/manifest.yml:544]: Do you think it would be useful to also add aws-cloudwatch as an option for the inputs here?

Comment (legoguy1000) [reply packages/aws/manifest.yml:544]: Looking at the docs CloudFront only sends logs to S3, https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/logging.html

File: packages/aws/_dev/build/docs/cloudfront.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# CloudFront
+
+## Logs
+
+The `cloudfront` dataset collects access logs from AWS CloudFront. CloudFront standard logs provide detailed records about every request that’s made to a distribution. These logs are useful for many scenarios, including security and access audits.
```
Comment (kaiyan-sheng) [inline packages/aws/_dev/build/docs/cloudfront.md:5]:
```suggestion
The `cloudfront` dataset collects standard logs(also called access logs) from AWS CloudFront. CloudFront standard logs provide detailed records about every request that’s made to a distribution. These logs are useful for many scenarios, including security and access audits.
```

File: packages/aws/manifest.yml (hunk 2 above)
Comment (kaiyan-sheng) [reply packages/aws/manifest.yml:544]: Cool thanks!! That makes things easier here 😬

**Comment by legoguy1000** [comment]
@kaiyan-sheng I believe this should be ready for review

**Review by kaiyan-sheng — APPROVED** [review]
LGTM! Thanks for adding this!

**Pull request merged**